typescript/sdk/src/deploy/HyperlaneAppChecker.ts

```diff
@@ -89,6 +93,10 @@ export abstract class HyperlaneAppChecker<
     const provider = this.multiProvider.getProvider(chain);
     const contracts = this.app.getContracts(chain);
 
+    const proxyAdminContract = proxyAdminAddress
```

Comment:
could this just be
```
const expectedProxyAdminContract = ProxyAdmin__factory.connect(expectedAdmin, provider)
```
So that we don't need to have this "prioritize proxyAdminAddress otherwise use the proxyAdmin contract from app` logic in two places